ksql-engine/src/main/java/io/confluent/ksql/topic/TopicDeleteInjector.java

```diff
@@ -93,9 +91,7 @@ public TopicDeleteInjector(
 
     if (source != null) {
       try {
-        ExecutorUtil.executeWithRetries(
-            () -> topicClient.deleteTopics(ImmutableList.of(source.getKafkaTopicName())),
-            RetryBehaviour.ALWAYS);
+        topicClient.deleteTopics(ImmutableList.of(source.getKafkaTopicName()));
```

Comment:
why have we removed the retries here?  Are there not still cases where we can get retryable exceptions here, e.g. on network issues, that are retryable?

Reply:
There was a delay when deleting the topic if the delete.topic.enable was false. I was thinking on not retrying if a `TopicDeletionDisabledException` was thrown, but I noticed that we weren't using this retry mechanism on the `TopicCreateInjector`, so thought this was just a copy/paste from the `ClusterTerminator` class.

However, I just noticed that this retry method is called in the `KafkaTopicClientImpl` instead. I will return it back, or better I will try to call it from the `deleteTopics` instead so we don't have to call retry anywhere else.